pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "numpy==1.14.5", "cython==0.29.0"]
```

Comment:
I have pinned numpy 1.14.5 as it's the first numpy version with py37 wheels on pypi. I have no idea what a good Cython pin is, there is no reason why we couldn't go very new here if we needed to.

Reply:
This works for python 3.8?

Reply:
I think the strategy is generally to choose the minimum NumPy that is both compatible with scikit-image and for which wheels exist. That is why SciPy has all of the following cases:

https://github.com/scipy/scipy/blob/6644a54ff862e4d90d7f9bc43ece9b0b1f681861/pyproject.toml#L5-L13

Reply:
Also, I think Cython >= 0.29.13 is the minimum compatible with Python 3.8.

Reply:
yes:

```
$ python --version
Python 3.8.0
$ pip freeze
pkg-resources==0.0.0
$ pip install .
```
numpy is now availible as a wheel for 3.8 but Cython 0.29.0 isn't, so I assume it has compiled it from source.

Reply:
I know that this build isolation used to rely on wheels existing, but it seems from this experiment that that is no longer the case. I wonder if I can find a pip reference for that somewhere.